manifest.yml

```diff
@@ -3,14 +3,10 @@ inputs:
   - name: checkPaths
     default: ['/']
     required: false
-  - name: debugMode
-    default: false
   - name: failWithIssues
     default: true
   - name: ignoreDirectories
     default: []
     required: false
   - name: standard
     default: 'WCAG2AA'
-  - name: testMode
-    default: false
```

Comment:
Note: this is a breaking change, so this would require a major release (https://github.com/netlify/plugins/blob/main/docs/versioning.md#breaking-changes) (which is ok).